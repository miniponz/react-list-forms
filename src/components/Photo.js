import React from 'react';
import PropTypes from 'prop-types';

function Photo({ url }) {
  return (
    <div>
      <img src={url}></img>
    </div>
  );
}

Photo.propTypes = {
  url: PropTypes.string.isRequired
};

export default Photo;
