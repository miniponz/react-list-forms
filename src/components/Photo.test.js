import React from 'react';
import { shallow } from 'enzyme';
import Photo  from './Photo';

describe('photo component', () => {
  it('renders photo', () => {
    const wrapper = shallow(<Photo image={'url'}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
