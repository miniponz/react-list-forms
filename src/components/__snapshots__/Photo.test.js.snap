// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`photo component renders photo 1`] = `
<div>
  <img />
</div>
`;
